subworkflows/local/vcf_impute_glimpse2/main.nf

```diff
@@ -23,6 +26,17 @@ workflow VCF_IMPUTE_GLIMPSE2 {
     // Add chr as key to input
     ch_input = ch_input.map{meta, bam, bai -> return[['chr': meta.chr], meta, bam, bai]}
 
+    // Fix panel (AC/AN INFO fields in VCF are inconsistent with GT field)
+    VCFLIB_VCFFIXUP(ch_panel)
```

Comment:
Is it necessary to run it each time ?
Can we detect it before running it ?

Reply:
I don't know if it takes a lot of time or not.

Reply:
We could fix the panel once in the `panelprep` step only.